packages/uni_app/lib/view/login/login.dart

```diff
@@ -197,67 +197,128 @@ class LoginPageViewState extends State<LoginPageView>
     final queryData = MediaQuery.of(context);
 
     return Theme(
-      data: applicationLightTheme.copyWith(
-        textSelectionTheme: const TextSelectionThemeData(
-          cursorColor: Colors.white,
-          selectionHandleColor: Colors.white,
+        data: applicationLightTheme.copyWith(
+          textSelectionTheme: const TextSelectionThemeData(
+            cursorColor: Colors.white,
+            selectionHandleColor: Colors.white,
+          ),
         ),
-      ),
-      child: Builder(
-        builder: (context) => Scaffold(
-          resizeToAvoidBottomInset: false,
-          backgroundColor: darkRed,
-          body: BackButtonExitWrapper(
-            child: Padding(
-              padding: EdgeInsets.only(
-                left: queryData.size.width / 8,
-                right: queryData.size.width / 8,
-              ),
-              child: Column(
+        child: Builder(
+          builder: (context) => Scaffold(
+            resizeToAvoidBottomInset: false,
+            backgroundColor: const Color(0xFF280709),
+            body: BackButtonExitWrapper(
+              child: Stack(
                 children: [
-                  SizedBox(height: queryData.size.height / 20),
-                  SizedBox(
-                    width: 100,
-                    // TODO(thePeras): Divide into two svgs to add color
-                    child: SvgPicture.asset(
-                      'assets/images/logo_dark.svg',
-                      colorFilter: const ColorFilter.mode(
-                        Colors.white,
-                        BlendMode.srcIn,
+                  Center(
+                    child: Align(
+                      alignment: const Alignment(0, -0.4),
+                      child: Hero(
+                        tag: 'logo',
+                        flightShuttleBuilder: (
+                          flightContext,
+                          animation,
+                          flightDirection,
+                          fromHeroContext,
+                          toHeroContext,
+                        ) {
+                          return ScaleTransition(
```

Comment:
What's the purpose of this transition? Can you upload a "demo"?

Reply:
I sent it as a video in the issue page, but here it is:
https://github.com/user-attachments/assets/12611925-2dc3-45bb-b13a-136a582c9b23